uefi-logger/src/writer.rs

```diff
@@ -1,4 +1,5 @@
 use super::Output;
+use uefi::{ucs2,Status,Result};
```

Comment:
Minor issue: should be space after each comma. `{ucs2, Status, Result}`, just like above.